server/server_test.go

```diff
@@ -273,20 +273,24 @@ func TestMultiRangeScanDeleteRange(t *testing.T) {
 	ds := kv.NewDistSender(&kv.DistSenderContext{Clock: s.Clock()}, s.Gossip())
 	tds := kv.NewTxnCoordSender(ds, s.Clock(), testContext.Linearizable, s.stopper)
 
-	if err := s.node.db.Call(proto.AdminSplit,
-		&proto.AdminSplitRequest{
+	if err := s.node.db.Run(&client.Call{
+		Args: &proto.AdminSplitRequest{
 			RequestHeader: proto.RequestHeader{
 				Key: proto.Key("m"),
 			},
 			SplitKey: proto.Key("m"),
-		}, &proto.AdminSplitResponse{}); err != nil {
+		},
+		Reply: &proto.AdminSplitResponse{}}); err != nil {
 		t.Fatal(err)
 	}
 	writes := []proto.Key{proto.Key("a"), proto.Key("z")}
 	get := &client.Call{
-		Method: proto.Get,
-		Args:   proto.GetArgs(writes[0]),
-		Reply:  &proto.GetResponse{},
+		Args: &proto.GetRequest{
```

Comment:
No reason. I was making this changes incrementally and changing hundreds of call sites. Just missed a few. I've given myself a TODO to revisit all these places and normalize the client usage. Even before this change many of the client calls were not idiomatic.